extensions/vscode/src/multiStepInputs/newDeployment.ts

```diff
@@ -267,7 +267,7 @@ export async function newDeployment(
 
   let entryPointListItems: QuickPickItemWithIndex[] = [];
   let inspectionResults: ConfigurationInspectionResult[] = [];
-  let contentRecordNames: string[] = [];
+  let contentRecordNames = new Map<string, string[]>();
```

Comment:
switch to a Map here so we can have easy filtering of configurations which are applicable to a projectDir where the entrypoints and deployment is being created.

Reply:
I wonder if it would be helpful in other inputs to have a util class for names to add them, use the `projectDir` to gather them and all that. Something to consider if we repeat the logic elsewhere.